### PR TITLE
[MB-11806] secondary addresses displayed in move history table

### PR DESCRIPTION
**Pull request by mmayo-truss**
## Setup to Run Your Code

<details>
<summary>💻 You will need to use two separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1.  Access the milmove app as a service member.
2. Go through the process of creating a move (Make sure you use the correct GBLOC, Fort McCoy is a duty station that will show up easily for our normal local dev patterns)
3. When you create the shipment make sure to add both secondary pickup and secondary delivery addresses. 
4. 🛑 Don't submit the move yet 
5. Go back and edit the secondary pickup addresses 
6. Signing and submit the move.
7. Log into the office app as a services counselor and view the move history for the move just created.
8. you should see the secondary addresses displayed as their own rows on the move history table.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Have the Jira acceptance criteria been met for this change?

## Screenshots
BEFORE:
image

AFTER:
image

### CODE REVIEW

File: src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.test.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('when given an mto shipment insert with address table history record',
   it('correctly matches the insert mto shipment address event for pickup addresses', () => {
     const result = getTemplate(item);
     expect(result).toMatchObject(e);
-    // expect to have formatted the adddresses correctly
```
Comment (kctruss) [inline src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.test.js:24]: **praise:** Nice catch on fixing this spelling 😍

File: src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.js [hunk 2]
```diff
@@ -13,13 +14,7 @@ export default {
...
 
     const addressType = context.filter((contextObject) => contextObject.address_type)[0].address_type;
-
-    let addressLabel = '';
-    if (addressType === 'pickupAddress') {
-      addressLabel = 'pickup_address';
-    } else if (addressType === 'destinationAddress') {
-      addressLabel = 'destination_address';
-    }
+    const addressLabel = AddressTypes[addressType];
```
Comment (michaelvaldes-truss) [inline src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.js:17]: **praise:** nice, this is way cleaner

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm! nice cleanup of the address types

**Review by kctruss — APPROVED** [review]
Nice job on this!

File: src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.js (hunk 2 above)
Comment (kctruss) [reply src/constants/MoveHistory/EventTemplates/createMTOShipmentAddresses.js:17]: 💯 Agreed, this is really nice

**Review by juliemoon — APPROVED** [review]
Hi @mmayo-truss thanks for doing this and answering questions I had regarding this PR. The purpose is to show creation and updates to shipments and this PR does that. The data the customer inputs in the shipments section shows in the move history page. There is a funny ordering issue where instead of showing created shipment first the updated shipment event shows up first. This is something that can be addressed in another ticket.

**Pull request merged**